lib/ansible/modules/network/nxos/nxos_ntp.py

```diff
@@ -164,15 +164,16 @@ def get_ntp_source(module):
     output = execute_show_command(command, module, command_type='cli_show_ascii')
 
     if output:
-        try:
-            if 'interface' in output[0]:
-                source_type = 'source-interface'
-            else:
-                source_type = 'source'
-            source = output[0].split()[2].lower()
-        except AttributeError:
-            source_type = None
-            source = None
+        if output[0]:
```

Comment:
Take this out and replace the `except` line to `except (AttributeError, TypeError)`